dsa/index.js

```diff
@@ -0,0 +1,91 @@
+const arr = [1, 2, 3, "a", 5, 6, "a", 8, 9];
+
+// Check that the arr has all unique entries
+
+//Time complexity
```

Comment:
Hey this look good, but can we please use JSDoc here. That has been our standard in the code base.

Reply:
Hey man. Thanks for the advice. I will do that now.